Source/Core/InputCommon/ControllerEmu/StickGate.cpp

```diff
@@ -285,6 +286,9 @@ ReshapableInput::ReshapeData ReshapableInput::Reshape(ControlState x, ControlSta
   x -= m_center.x;
   y -= m_center.y;
 
+  // We run this even if both x and y will be zero.
+  // The angle value will be random but dist will stay 0
+
   // TODO: make the AtAngle functions work with negative angles:
   ControlState angle = std::atan2(y, x) + MathUtil::TAU;
```

Comment:
I don't think it's entirely accurate to say the angle value will be _random_; [it's actually specified](https://en.cppreference.com/w/cpp/numeric/math/atan2#Error_handling) to be either +0, -0, +Pi, or -Pi depending on the sign of x and y.  I'm not quite sure what phrasing would be best.

Reply:
Maybe "`std::atan2(0, 0)` returns a valid non-NaN value, but the exact value (which depends on the signs of x and y) does not matter here as dist is zero"?